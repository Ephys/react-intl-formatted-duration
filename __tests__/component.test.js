import React from 'react';
import { default as DurationMessage, EXTENDED_FORMAT } from '../index';
import { mountWithIntl } from 'enzyme-react-intl';

function Text(props) {
  return <x-text {...props} />;
}

function Value(props) {
  return <x-value {...props} />;
}

describe('Component', () => {
  it('generates a stable dom', () => {
    const component = mountWithIntl(
      <DurationMessage seconds={61} format={EXTENDED_FORMAT} textComponent={Text} valueComponent={Value} />,
    );

    expect(component.html()).toMatchSnapshot();
  });
});
