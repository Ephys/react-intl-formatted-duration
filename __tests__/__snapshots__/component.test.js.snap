// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component generates a stable dom 1`] = `"<x-text><x-value>1</x-value> <x-text>minute</x-text> <x-value>1</x-value> <x-text>second</x-text></x-text>"`;
